pkg/apis/networking/validation/validation_test.go

```diff
@@ -1018,6 +1018,32 @@ func TestValidateNetworkPolicy(t *testing.T) {
 				},
 			},
 		},
+		"invalid endport range defined": {
```

Comment:
Not going to hold this PR to it, but this is a great example of a place where some helper functions could go a LONG way.

E.g. this should be something like:

```
"invalid endport range defined": makeService(egress(port(intstr.FromInt(30000), 65537)))
```

Everything else seems like boilerplate.  Repeat that for every test case in this one test and delete hundreds of LOC

Reply:
will do in a follow-up, thanks Tim for the feedback